### ISSUE TITLE
API-request: ability to filter the last changed items on list endpoints

### ISSUE THREAD

**Issue by Kobo-one** [TheRestartProject/restarters.net#525]
Would it be possible to add a filter `updated_since` to the list endpoint. 
That way it would be possible to query the endpoints only based on data that changed instead of returning every item. 

This would be used by Statik to connect Restarters.net and Repair Connects. We store some of the group and event data to make device live logging possible. This is done using a periodic sync.

Possible endpoints: `/groups/network` and `/events/network`

**Comment by edwh** [comment]
I think the query parameter to triggering the filtering should be `updated_since`, but the field on the individual objects should be `updated_at`.

**Comment by ngm** [comment]
Hi @Kobo-one ,

The following endpoints should now be available to you: 

* https://restarters.net/api/v2/networks/<id>/groups
* https://restarters.net/api/v2/networks/<id>/events

Both endpoints have the following parameters available:

* updated_start
* updated_end

which can take an ISO8601 formatted date, e.g. 2020-01-01.  If either are left blank, no restriction to that end of the date range will be made.

**Comment by Kobo-one** [comment]
HI @ngm,

Thank you.
It seems there is a new parameter `id` in the query now. 
I used `https://restarters.net/api/v2/networks/` to find the correct network-id for our use case but noticed it returned multiple networks. 

I was able to get (network, group and event) data from other networks aswel. Is this expected behaviour?

**Comment by ngm** [comment]
Yes, we haven't chosen to restrict any of the information in v2 for now, given that none of it is personal or private.  Given what we discussed about ORDP earlier, it seems potentially possible that we'll expose all of this info in future for that purpose.   That said, it's possible we may decide otherwise, or to provide access  to everything for ORDP via a different 'firehose' endpoint, so don't necessarily rely on having access to other networks this way in future :)
